lib/Rides.dart

```diff
@@ -51,31 +52,30 @@ class RidesStateless extends StatelessWidget {
   Widget ridesInProgress(BuildContext context) {
     return Container(
         child: Column(children: [
-      Padding(
-        padding: EdgeInsets.symmetric(horizontal: 16),
-        child: RideGroupTitle('In Progress', currentRides.length),
-      ),
-      GridView.count(
-        padding: EdgeInsets.only(top: 24, bottom: 32, left: 16, right: 16),
-        mainAxisSpacing: 16,
-        crossAxisSpacing: 16,
-        physics: NeverScrollableScrollPhysics(),
-        crossAxisCount: 2,
-        childAspectRatio: 0.8,
-        shrinkWrap: true,
-        children: currentRides
-            .asMap()
-            .map((i, ride) {
+          Padding(
+            padding: EdgeInsets.symmetric(horizontal: 16),
+            child: RideGroupTitle('In Progress', currentRides.length),
+          ),
+          GridView.count(
+            padding: EdgeInsets.only(top: 24, bottom: 32, left: 16, right: 16),
+            mainAxisSpacing: 16,
+            crossAxisSpacing: 16,
+            physics: NeverScrollableScrollPhysics(),
+            crossAxisCount: 2,
+            shrinkWrap: true,
+            children: currentRides
+                .asMap()
+                .map((i, ride) {
               Widget w = RideInProgressCard(Key(ride.id), ride,
                   selectedRides.contains(ride), selectCallback);
               if (i == 0)
                 w = MeasureRect(child: w, onChange: firstCurrentRideRectCb);
               return MapEntry(i, w);
             })
-            .values
-            .toList(),
-      )
-    ]));
+                .values
```

Comment:
Any reason for the indentation here?

Reply:
Discussed offline: likely VSCode vs Android Studio differences